src/EditorFeatures/TestUtilities/Diagnostics/AbstractDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -36,6 +36,102 @@ internal virtual (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderA
                 : CreateDiagnosticProviderAndFixer(workspace, parameters);
         }
 
+        [Fact]
+        public void TestSupportedDiagnosticsMessageTitle()
+        {
+            using (var workspace = new AdhocWorkspace())
+            {
+                DiagnosticAnalyzer diagnosticAnalyzer;
+                try
+                {
+                    diagnosticAnalyzer = CreateDiagnosticProviderAndFixer(workspace).Item1;
+                }
+                catch (NotSupportedException)
+                {
+                    return;
```

Comment:
when does this happen?

Reply:
➡️ 

https://github.com/dotnet/roslyn/blob/d95e0e6d31afba486804c301109f6b1ef2b6cbe2/src/EditorFeatures/VisualBasicTest/Diagnostics/AddImport/AddImportTests_NuGet.vb#L32-L34

Reply:
A comment might be wise here.

Reply:
➡️ Fixed in 15f74696d039687953d4b36c452494fac7c06499